pom.xml

```diff
@@ -258,7 +258,7 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-jar-plugin</artifactId>
-                        <version>3.2.2</version>
+                        <version>3.3.0</version>
```

Comment:
As we use this in two places (line258, and line113) would it make sense to have a parameter for the maven-jar-plugin version?

Reply:
I have changed it, hope you do not mind, just to close these PR-s fastly.